frontend/src/router/index.ts

```diff
@@ -110,6 +111,17 @@ const routes = [
         component: GrantDelegatedAdminView,
         beforeEnter: beforeEnterHandlers[routeItems.grantDelegatedAdmin.name],
     },
+    {
+        path: routeItems.myPermissions.path,
+        name: routeItems.myPermissions.name,
+        meta: {
+            requiresAuth: false,
```

Comment:
Hi Jota, we still need authentication, so requiresAuth means this page needs user login first, all our routes need login (landing page doesn't need)